### PR TITLE
[new release] extunix (0.4.0)

### PR DESCRIPTION
**Pull request by MisterDA**
Collection of thin bindings to various low-level system API

- Project page: <a href="https://github.com/ygrek/extunix">https://github.com/ygrek/extunix</a>

##### CHANGES:

* Remove top-level module names, modules should be accessed via
  ExtUnix top-level module.
* Import realpath from OCaml 4.13
* Support realpath on Windows

### CODE REVIEW

**Comment by mseri** [comment]
The following packages need an upper bound on extunix:

- [devkit.1.2 (failed: Unbound module ExtUnixAll)](https://opam.ci.ocaml.org/github/ocaml/opam-repository/commit/25d56435aee2453638d11ea5e8868acf6789a1dc/variant/opam-2.1,compilers,4.13,extunix.0.4.0,revdeps,devkit.1.2)
- [devkit.1.20210120 (failed: Unbound module ExtUnixAll)](https://opam.ci.ocaml.org/github/ocaml/opam-repository/commit/25d56435aee2453638d11ea5e8868acf6789a1dc/variant/opam-2.1,compilers,4.13,extunix.0.4.0,revdeps,devkit.1.20210120)
- [devkit.1.3 (failed: Unbound module ExtUnixAll)](https://opam.ci.ocaml.org/github/ocaml/opam-repository/commit/25d56435aee2453638d11ea5e8868acf6789a1dc/variant/opam-2.1,compilers,4.13,extunix.0.4.0,revdeps,devkit.1.3)

**Comment by ygrek** [comment]
idk what is the policy but I am ok with just deleting all except the latest 1.20210517

**Comment by mseri** [comment]
The policy is to add the upper bounds to the relevant opam files and leave them in the opam repository. Can you do it? I will not be able to do it before tuesday

**Comment by mseri** [comment]
Thanks @kit-ty-kate

File: packages/devkit/devkit.1.20210517/opam [hunk 1]
```diff
@@ -20,7 +20,7 @@ depends: [
   "ocurl" {>= "0.7.2"}
   "ocamlnet"
   "pcre"
-  "extunix" {>= "0.1.4"}
+  "extunix" {>= "0.1.4" & < "0.4.0"}
```
Comment (ygrek) [inline packages/devkit/devkit.1.20210517/opam:23]: this one is not needed, should be compatible and CI agrees iiuc

Comment (kit-ty-kate) [reply packages/devkit/devkit.1.20210517/opam]: my bad, reverted

**Comment by kit-ty-kate** [comment]
Thanks!

**Pull request merged**